src/modules/site-v2/templates/_scripts/utils.js

```diff
@@ -92,10 +92,7 @@ function force_download(e) {
       // If everything succeeds, all future clicks will use the modified href
       // in the existing anchor element. If something fails, all clicks should
       // restart the blob download process, which is is acceptable.
-      const a = document.createElement('a');
-      a.href = el.href;
-      a.download = el.download || 'download';
-      a.click();
+      download_from_href(el.href, el.download || 'download');
```

Comment:
consider renaming `download_from_url()`, since semantically, it could come from a non `href` prop.